src/main/java/se/kth/sda/skeleton/posts/exeption/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package se.kth.sda.skeleton.posts.exeption;
```

Comment:
Can you move this from posts package to:
``package se.kth.sda.skeleton.exeption``
As we will use that exception in other controllers, so it's better not to have it related to posts only.

Reply:
"ex**c**eption"

Reply:
hahah! I can never get that one right

Reply:
Renamed the package and moved to root.